mlflow/environment_variables.py

```diff
@@ -76,6 +76,11 @@ def get(self):
         return lowercased in ["true", "1"]
 
 
+# define here to avoid circular import of databricks_utils
+def is_in_databricks_runtime():
+    return "DATABRICKS_RUNTIME_VERSION" in os.environ
```

Comment:
Do we need this?

Reply:
It's defined in databricks_utils, or we just do not define a function and use `"DATABRICKS_RUNTIME_VERSION" in os.environ` directly here?